rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
I removed with_flattened because it expands strings as variables, so "swift" was getting replaced with the swift config dict - see above comment